google-api-client/pom.xml

```diff
@@ -117,6 +117,11 @@
     </resources>
   </build>
   <dependencies>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
```

Comment:
This library already depends on jackson2 and we don't need 2 JSON parsing/serialization libraries at the same time. We should pick one and stick to it.

Reply:
removed gson dependency